### PR TITLE
dkg_submit

### PR DESCRIPTION
**Pull request by Aasifj2**
I have updated the dkgsubmit event.

### CODE REVIEW

**Review by roshgregory — APPROVED** [review]
good stuff, lets test it

**Comment by roshgregory** [comment]
we'll have to change around the yoda error, since we're hardcoding addresses towards the end

File: pkg/chain/ethereum/beacon.go [hunk 1]
```diff
@@ -354,9 +368,169 @@ func (bc *BeaconChain) SubmitDKGResult(
...
+	if err != nil {
+		return err
+	} else {
+		fmt.Println(tx)
+		return nil
+	}
+
 }
 
+func hash_members(members []uint32) ([32]byte, error) {
```
Comment (sharmioshin) [inline pkg/chain/ethereum/beacon.go:397]: Add information about the functionality of this function

File: pkg/chain/ethereum/beacon.go [hunk 2]
```diff
@@ -354,9 +368,169 @@ func (bc *BeaconChain) SubmitDKGResult(
...
+
+	bytes, err := abi.Arguments{{Type: uint32SliceType}}.Pack(members)
+	if err != nil {
+		return [32]byte{}, err
+	}
+
+	return crypto.Keccak256Hash(bytes), nil
+}
+
+func convertSignaturesToChainFormat2(
```
Comment (sharmioshin) [inline pkg/chain/ethereum/beacon.go:411]: Are we pushing the signature to the chain?

**Review by sharmioshin — COMMENTED** [review]
Seems good. Run and test the rsults

**Pull request merged**